### ISSUE TITLE
Docu update for SDKization

### ISSUE THREAD

**Issue by g2flyer** [hyperledger-labs/fabric-private-chaincode#96]
Bundle all docu update related to the SDKization refactoring and related (e.g., the removal of the explicit docker switcheroo) to after the corresponding code changes are done

**Comment by mbrandenburger** [comment]
- [ ] ECC refactoring #69 

- [ ] remove switcheroo #97

**Comment by mbrandenburger** [comment]
Fix also _Time to get the code_ section.

```
$ git clone https://github.com/hyperledger-labs/fabric-private-chaincode.git $GOPATH/src/hyperledger-labs/fabric-private-chaincode
```

```
$ git clone https://github.com/hyperledger-labs/fabric-private-chaincode.git $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode
```

**Comment by mbrandenburger** [comment]
Consolidate READMEs. With the new build system we don't need extra README in each component folder that explains how to build the component. I suggest that we combine them in a single README and only provide extra information if needed.

**Comment by mbrandenburger** [comment]
content for new section FAQ:

- `sudo make` solves issues with misconfigured docker environments but changes `$GOPATH`

https://docs.docker.com/install/linux/linux-postinstall/
https://stackoverflow.com/questions/55843277/cant-run-go-program-using-sudo-command

**Comment by mbrandenburger** [comment]
- Maybe move "Behind a proxy" to FAQ section or closer to IAS section

**Comment by blockchaindemo** [comment]
Add:    sudo snap install --stable --classic protobuf
in Get NanoPB section

**Comment by g2flyer** [comment]
we should  also a simple software development section on the FPC programming model (shim.h et al ...)

**Comment by mbrandenburger** [comment]
> we should also a simple software development section on the FPC programming model (shim.h et al ...)
> 
> We definitely also should document that folks have to use the wrapper scripts from fabric/bin and point out the utility scripts fabric/bin/ledger_init & fabric/bin/ledger_shutdown if users want to interactively play with FPC ..

Let's address this briefly in #129 and provide concise documentation on the programming model later.

**Comment by g2flyer** [comment]
addressed mostly in PR #142 (issue #129) and PR #137 (with increased docu inside shim.h)
